### PR TITLE
Several stabilization changes to get us closer to a playable build, including:

### PR DESCRIPTION
**Pull request by Sunrunner37**
image

Entities are mostly spawning - still some slowness in starting chunks though.  It also looks like simulation ownership system is working fine.  This PR does have some benign unauthenticated packets due to moving the dynamic patching location - we can work through these with suppression when we get to them. 

* Changed dynamic patching hook to be earlier on.  This allows patches that require early initialization to function (such as uGUI_PDA)
* Fixed several bugs in entity spawning async transformation
* Fixes to allow users to spawn alongside their remote vitals
* Prevented PDA from erroring out initial syncs with multiple players
* Added a new patch to prevent escape pod supplies from spawning
* Small tweak to logger to only log a warning instead of throwing when attempting to re-initialize.
* Fixed a bug preventing us from finding the remote player's inventory.

### CODE REVIEW

**Review by dartasen — APPROVED** [review]

**Review by Coding-Hen — APPROVED** [review]

**Comment by Jannify** [comment]
I encountered a few errors:

- Changing the patch trigger point errors out `CyclopsHelmHUDManager_Start_Patch` on the cyclops prototype scene which is loaded additive
- I got a NRE at `RemotePlayerVitals.CreateForPlayer`. I think this was also present before the SN update
- `uGUI_PlayerListTab` is erroring out hard when a second player joins. if you open the PDA it displays nothing and you are stuck in it forever!
- Strangely the Flare threw an NRE error in Awake on pickup

Would definitely want to have the PDA fixed, rest is optional/can be done later.

File: NitroxPatcher/Patches/Dynamic/SpawnEscapePodSupplies_OnNewBorn_Patch.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+using System.Reflection;
+using HarmonyLib;
+using NitroxModel.Helper;
+using UWE;
+
+/// <summary>
+/// Don't allow the game to spawn initial supplies in the escape pod.
+/// </summary>
+namespace NitroxPatcher.Patches.Dynamic
+{
```
Comment (Jannify) [inline NitroxPatcher/Patches/Dynamic/SpawnEscapePodSupplies_OnNewBorn_Patch.cs:10]: Would prefer file scoped namespaces

File: NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs [hunk 2]
```diff
@@ -58,10 +58,10 @@ public IEnumerator SpawnAsync(Entity entity, Optional<GameObject> parent, Entity
...
         {
             GameObject prefab = null;
-            if (PrefabDatabase.TryGetPrefabFilename(classId, out string filename))
-            {
-                prefab = Resources.Load<GameObject>(filename);
-            }
+
+            IPrefabRequest prefabCoroutine = PrefabDatabase.GetPrefabAsync(classId);
+            yield return prefabCoroutine;
+            prefabCoroutine.TryGetPrefab(out prefab);
```
Comment (Jannify) [inline NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs:64]: Would move variable declaration inside the TryGetPrefab()

**Review by Jannify — CHANGES_REQUESTED** [review]

**Comment by Sunrunner37** [comment]
Fixed the PDA issue.  Will address cyclops issues as a part of a larger vehicle refactoring in next PR.

File: NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs (hunk 2 above)
Comment (Sunrunner37) [reply NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs]: Done

File: NitroxPatcher/Patches/Dynamic/SpawnEscapePodSupplies_OnNewBorn_Patch.cs (hunk 1 above)
Comment (Sunrunner37) [reply NitroxPatcher/Patches/Dynamic/SpawnEscapePodSupplies_OnNewBorn_Patch.cs]: Done

**Review by Jannify — APPROVED** [review]

**Pull request merged**